sections/getting-started.tex

```diff
@@ -100,7 +100,7 @@ \subsection{Workshop git project}
   \begin{lstlisting}
 $ (*\textcolor[HTML]{0000AA}{cd gitmoon}*)
 $ (*\textcolor[HTML]{0000AA}{ls}*)
-assets    moon.html    moon.js
+assets    assets.js    moon.html    moon.js
```

Comment:
off-topic: is this `assets` folder still relevant when we have the `assets.js` ?

Reply:
I think it is not used in any way at the moment but I opted to keep it because it contains the images in a non base64 form.